ca/certificates.go

```diff
@@ -381,9 +387,24 @@ func NewRootCA(certBytes, keyBytes []byte, certExpiry time.Duration) (RootCA, er
 	// Calculate the digest for our Root CA bundle
 	digest := digest.FromBytes(certBytes)
 
+	// TODO(cyli): We currently do not yet support arbitrary chains of intermediates (e.g. the case of an offline root, with the CA signing with
```

Comment:
@diogomonica @aaronlehmann I can probably also handle this case but it may be better to do that in another PR, because there could be a bunch of changes (to distinguishing the root certs the RootCA should trust, vs the signing cert).